### DISCUSSION TITLE
Global locale.

### DISCUSSION THREAD

**Discussion by borrrsch** [requarks/wiki#3368] [Help / Questions]
OS: Arch Linux 5.6.2-arch1-2 x64
Wiki.js version: 2.2.51
Database engine: postgres 12.2

I set locale from UI, page opens correctly, but then I click on link to that page, locale switches by default, that is different. How can I switch to locale globally?

**Comment by NGPixel** [comment]
Did you add the locale to the link? If you want to link to another page that isn't the main locale, you need to include it in the path (e.g. `/fr/some/path`)

**Comment by borrrsch** [comment]
Navigation link is a relative link. "/en /ru" adding automatically. Opening page language version depends on locale. But locale switches by default (/ru) every time then I click on the nav link. I want to keep locale suffix /en until I switch it manually again.
